internal/testing/processor/testdata/github-review.json

```diff
@@ -0,0 +1,20 @@
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [
+      {
+        "name": "git://github.com/kubernetes/kubernetes", 
+        "digest": {"sha1": "5835544ca568b757a8ecae5c153f317e5736700e"}
+      }
+  ],
+  "predicateType": "http://in-toto.io/attestation/certify",
```

Comment:
One more minor change here, similar to `https://slsa.dev/provenance/v0.2` can we make this something like `https://in-toto.io/attestation/certify/v0.1`?